### PR TITLE
Fix generated deployment parameter not respecting `None` as a default value

### PR DESCRIPTION
**Pull request by pleek91**
# Description
Found a couple places where we were checking for nullish values when we should be only checking for undefined. This would mean that `None` values were not treated as real values.

### CODE REVIEW

File: src/schemas/components/SchemaFormProperties.vue [hunk 1]
```diff
@@ -79,7 +79,7 @@
   }
 
   function getValue(propertyKey: string): unknown {
-    return props.values?.[propertyKey] ?? undefined
+    return props.values?.[propertyKey]
```
Comment (pleek91) [inline src/schemas/components/SchemaFormProperties.vue:82]: Since `null` (none) is a valid value we shouldn't coerce it to undefined.

File: src/schemas/components/SchemaFormPropertyAnyOf.vue [hunk 2]
```diff
@@ -96,12 +96,14 @@
 
   const property = computed(() => {
     const selectedProperty = props.property.anyOf[selectedPropertyIndex.value]
+    // eslint-disable-next-line no-unused-vars
+    const { anyOf, ...property } = props.property
```
Comment (pleek91) [inline src/schemas/components/SchemaFormPropertyAnyOf.vue:100]: unrelated change that strips the `anyOf` from the schema for an individual `anyOf` property. Wasn't hurting anything currently but would cause issues with validating a single property.

**Review by znicholasbrown — APPROVED** [review]
